pkg/cmd/cli/cmd/expose.go

```diff
@@ -50,6 +56,8 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 	// when validating the use of it (invalid for routes)
 	cmd.Flags().Set("protocol", "")
 	cmd.Flag("protocol").DefValue = ""
+	// Set target-port usage to note that it can be used for routes, too.
+	cmd.Flag("target-port").Usage = "Number or name for the port on the container that the route or service should direct traffic to. Optional."
```

Comment:
I don't think we can write a single usage string that makes sense for all the things `expose` handles. Even in the route case, explaining target-port is a little crazy... a name is the service port name, a number is a container port number.